core/chains/evm/gas/rollups/op_l1_oracle_test.go

```diff
@@ -111,6 +111,54 @@ func TestOPL1Oracle_ReadV1GasPrice(t *testing.T) {
 	}
 }
 
+func TestOPL1Oracle_ReadMantleGasPrice(t *testing.T) {
+	l1BaseFee := big.NewInt(100)
+	tokenRatio := big.NewInt(40)
+	oracleAddress := common.HexToAddress("0x1234").String()
+
+	t.Parallel()
+	t.Run("correctly fetches gas price if chain is Mantle", func(t *testing.T) {
```

Comment:
Could you also add some simple test cases for RPC errors? I think we have some for Ecotone and Fjord for reference.

Reply:
Thanks, I just addressed this in the latest commit